src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -332,3 +332,46 @@ def from_argilla(
             questions=questions,
             guidelines=existing_dataset.guidelines or None,
         )
+
+    @classmethod
+    def list(cls: Type["FeedbackDataset"], workspace: Optional[str] = None) -> List[RemoteFeedbackDataset]:
+        """Lists the `FeedbackDataset`s pushed to Argilla.
+
+        Note that you may need to `rg.init(...)` with your Argilla credentials before
+        calling this function, otherwise, the default http://localhost:6900 will be used,
+        which will fail if Argilla is not deployed locally.
+
+        Args:
+            workspace: the workspace where to list the datasets from. Defaults to `None`
+                which means that no workspace filtering will be applied.
```

Comment:
With Google Docstring style I always prefer to have the "Defaults to ..." at the end of the arg description. Feel free to ignore this.

```suggestion
            workspace: the workspace where to list the datasets from. If not provided,
                then the workspace filtering won't be applied. Defaults to `None`.
```